### DISCUSSION TITLE
will you add teammate name display class color fucntion  and teammate traget frame ?

### DISCUSSION THREAD

**Discussion by Jack8713** [kaytotes/ImprovedBlizzardUI#183] [Ideas]
will you add teammate name display class color fucntion  and teammate traget frame ?

**Comment by kaytotes** [comment]
I'm not sure I understand what you mean by this? Could you add a screenshot of the Blizzard frames.

**Reply by Jack8713** [reply]
Party Frames  like 
https://gimg2.baidu.com/image_search/src=http%3A%2F%2Fimg.nga.178.com%2Fattachments%2Fmon_201909%2F05%2FifQ5-bi7gZ1sT3cSs4-f4.jpg&refer=http%3A%2F%2Fimg.nga.178.com&app=2002&size=f9999,10000&q=a80&n=0&g=0n&fmt=jpeg?sec=1626022190&t=66992c2004d7024cb089bbfe593c769b
